filament/include/filament/Engine.h

```diff
@@ -291,6 +291,14 @@ class UTILS_PUBLIC Engine {
          * Currently only respected by the Metal backend.
          */
         size_t textureUseAfterFreePoolSize = 0;
+
+        /*
+         * The number of eyes to render when stereoscopic rendering is enabled. Supported values are
+         * between 1 and CONFIG_MAX_STEREOSCOPIC_EYES (inclusive).
+         *
+         * @see View::setStereoscopicOptions
+         */
+        uint8_t stereoscopicEyeCount = 2;
```

Comment:
I checked `OVR_multiview` and it indeed needs to know the number of views in vertex shaders, so we do need to know this value when we create materials if we want to support it someday. This is unfortunate because if it wasn't for that, this setting could be in `View` (where it would make much more sense).
I think there should be a comment about that somewhere (in a cpp).

Reply:
Could we not use a spec constant with `OVR_multiview`?

Reply:
So we could use a spec-constant, but that needs to be known at material creation time because we can't update spec-constant afterwards (technically we can by invalidating the cache, but we need to flush this out more).
There is also a small hurdle which is that GLSL accepts const expressions, but ESSL doesn't :-(  So for GLES we'll have to hack a #define.

Reply:
> so we do need to know this value when we create materials if we want to support it someday

Do you mean the material author needs to know this value, or Filament needs to know this value when a material is built?